node/forest_libp2p/Cargo.toml

```diff
@@ -45,6 +45,8 @@ ipld_blockstore = "0.1"
 async-trait = "0.1"
 lazy_static = "1.4"
 
+futures_cbor_codec = { git = "https://github.com/najamelan/futures_cbor_codec", rev = "de08e31530513f993cbf5efef7311ac5194b2256" }
```

Comment:
Did the most recent release, `0.2` not work?

Reply:
Is it because of the version bump from the commit after the release?

Reply:
Yeah. The 0.2 release is using futures-codec 0.3 whereas I wanted to use 0.4.